guides/functional_api.py

```diff
@@ -390,7 +390,10 @@ def get_model():
         "priority": keras.losses.BinaryCrossentropy(from_logits=True),
         "department": keras.losses.CategoricalCrossentropy(from_logits=True),
     },
-    loss_weights=[1.0, 0.2],
+    loss_weights={
+        "priority": 1.0,
+        "department": 0.2,
+    },
```

Comment:
I was wondering how it lines up the outputs to the losses given that the outputs are defined as: `outputs=[priority_pred, department_pred]`

But now I see that it's using the layer names. 

Should there be an example using `outputs={'priority': priority_pred, 'department': department_pred}`? I find that form is always clearer for the user of a multiple output model.

Reply:
Updated the description text so it's clearer where these names come from.

I don't know what's better, specifying the names in the layer or in the model.fit() call. I leave that to you.

Reply:
Regardless of that Idea, I think this is a good change. Let's get it merged.